format/index.md

```diff
@@ -1453,6 +1481,14 @@ constraints (such as a uniqueness constraint on a property other than `id`).
 A server **MUST** return `409 Conflict` when processing a `PATCH` request in
 which the resource object's `type` and `id` do not match the server's endpoint.
 
+A server **SHOULD** return `409 Conflict` when processing a `PATCH` request in
+which the resource object's `revision` does not match the server's endpoint
+and the server was *unable to resolve* the `revision` mismatch for updating
+resources with the same `type` and `id` but mismatched `revision` members.
+
+> Note: Only applicable if a server supplied the **OPTIONAL** `revision` member
```

Comment:
Since it is not mandatory for the client to supply `revision` in a `PATCH`, and since `GET` before `PATCH` is not mandatory, I believe this should be changed to 

> Note: Only applicable if a client supplied the **OPTIONAL** `revision` member in the `PATCH` request.